neuralprophet/configure.py

```diff
@@ -397,12 +397,12 @@ def __post_init__(self):
             self.seasonality_local_reg = False
 
         # If seasonality_local_reg = True
-        if self.seasonality_local_reg == True:
+        if self.seasonality_local_reg:
```

Comment:
This also seems to come from another PR

Reply:
Also, this change will always override a set `seasonality_local_reg` and overwrite it to be `1` which does not appear intentional?

Reply:
No was trying to solve the flake8 errors - doesn't make any difference anyways